webapp/src/view/home/components/AppCard.tsx

```
@@ -198,58 +230,87 @@ export default function AppCard({
+                        sx: {
+                          backgroundColor: "#fff",
+                          color: "#333",
+                          border: "1px solid #e6e6e6",
+                          borderRadius: 1,
+                          boxShadow: "0 2px 8px rgba(0,0,0,0.1)",
+                          flex: "wrap",
+                          maxWidth: "250px"
+                        },
+                      },
+                    }}
+                  >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)